src/Forms/Prism.Forms/Navigation/INavigationParametersInternal.cs

```diff
@@ -1,11 +1,30 @@
 ﻿namespace Prism.Navigation
 {
+    /// <summary>
+    /// Internal - Provides a way for the <see cref="INavigationService"/> to pass parameteres during navigation.
+    /// </summary>
     public interface INavigationParametersInternal
     {
+        /// <summary>
+        /// Adds the key and value to the KeyValuePair<string,object> collection
```

Comment:
this is invalid markup... second we don't really want to reference the KeyValuePair's outright... just refer to them as parameters